### DISCUSSION TITLE
Self Test

### DISCUSSION THREAD

**Discussion by Iceymidgit** [sanni/cartreader#765] [Need help]
G'day i keep getting an error on selftest A3 + A5 but i have resoldered everything i even pulled off cart slots and still same error so i re soldered the bottome again this is the 6 slot version top and bottom .

i can't see any joining solder joins

**Comment by sanni** [comment]
Can you try without the six slot adapter attached? I would guess the error persists without it.

This is the only location where A3 and A5 are next to each other(both on the main PCB and the Mega Pro):

![yxvgydgydsgysd](https://user-images.githubusercontent.com/19169494/230713423-73ad7b47-0ed4-4d53-b15c-d4e04a99867f.jpg)

If you have a multimeter you can also measure the resistance between the two pins to get a confirmation.

**Reply by Iceymidgit** [reply]
Thanks abit of wire was stuck under pins
